sdk-tests/src/test/java/io/dapr/it/pubsub/http/PubSubIT.java

```diff
@@ -30,6 +30,7 @@
   //Number of messages to be sent: 10
   private static final int NUM_MESSAGES = 10;
 
+  private static final String PUBSUB_NAME = "messagebus";
```

Comment:
You may want to make this public so you don't have to write "messagebus" each time in other files like SubscriberController.java.